<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Photo.qrc">
    <normaloff>:/new/prefix1/photo/icon.png</normaloff>:/new/prefix1/photo/icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>751</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>290</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Agency FB&quot;;
background-image: url(:/new/prefix1/photo/color.png);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
font: 10pt &quot;幼圆&quot;;
color: rgb(21, 216, 224);
border:0.5px groove gray;border-radius:10px;padding:2px 4px;</string>
           </property>
           <property name="text">
            <string> 结束会话 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_addordelete">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
color: rgb(21, 216, 224);
border:0.5px groove gray;border-radius:10px;padding:2px 4px;
font: 10pt &quot;幼圆&quot;;</string>
           </property>
           <property name="text">
            <string> 添加/删除好友 </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_friInfo">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>250</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;幼圆&quot;;
background-image: url(:/new/prefix1/photo/color.png);
color: rgb(255, 255, 255);
border:0px groove gray；</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'幼圆'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>好友列表</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Photo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainInterface</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1014</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>1004</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainInterface</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>932</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
